utils/build/docker/nodejs/express4/iast/sources.js

```diff
@@ -78,6 +78,15 @@ function init (app, tracer) {
     res.send('OK')
   })
 
+  app.get('/iast/source/path_parameter/test/:table', (req, res) => {
+    try {
```

Comment:
To follow the pattern of the rest of source endpoints, as catching - or not - the error doesn't change the test result.

I've added try-catches to the all source endpoints in ts application that call to `readFileSync` to avoid having `ENOENT` in weblog logs.